lib/phoenix_playground/endpoint.ex

```diff
@@ -19,8 +19,20 @@ defmodule PhoenixPlayground.Endpoint do
   plug Plug.Static, from: {:phoenix, "priv/static"}, at: "/assets/phoenix"
   plug Plug.Static, from: {:phoenix_live_view, "priv/static"}, at: "/assets/phoenix_live_view"
 
-  socket "/phoenix/live_reload/socket", Phoenix.LiveReloader.Socket
-  plug :run_live_reload
+  if Application.compile_env(:phoenix_playground, :live_reload, true) do
+    socket "/phoenix/live_reload/socket", Phoenix.LiveReloader.Socket
+    plug :run_live_reload
+
+    defp run_live_reload(conn, _options) do
+      if Application.get_env(:phoenix_playground, :live_reload) do
```

Comment:
we already have a check if live_reload is enabled on line 22, I'm curious if you intentionally left this check _here_ to allow changing the value at runtime somehow?

Reply:
I'll go through this, it was early on and I may have spiked this incorrectly.